chromium_src/chrome/installer/setup/uninstall.cc

```diff
@@ -71,6 +73,8 @@ InstallStatus UninstallProduct(const ModifyParams& modify_params,
                    << brave_vpn::GetBraveVpnHelperServiceName();
     }
   }
+  brave_vpn::ras::RemoveEntry(brave_vpn::GetBraveVPNConnectionName());
```

Comment:
remove works if vpn is connected state?

Reply:
Nice catch, added Disconnect call inside RemoveEntry, works well

Reply:
Q) `UninstallProduct()` method can call blocking call?

Reply:
sure